config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -407,7 +407,7 @@ presubmits:
         - '--node-test-args=--container-runtime=remote --container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service --non-masquerade-cidr=0.0.0.0/0" --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u\", \"crio\"]}"'
         - --node-tests=true
         - --provider=gce
-        - --test_args=--nodes=8 --focus="\[NodeConformance\]|\[NodeFeature:.+\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"
+        - --test_args=--nodes=8 --focus="\[NodeConformance\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"
```

Comment:
does not seem related to PR title.

Reply:
Thanks @BenTheElder for your reply. This is an existing presubmit job that we would like to reduce the scope by removing the `NodeFeatures` tests and focusing only on `NodeConformance` tests before adding to the `presubmits-kubernetes-nonblocking` tab.